.vscode/settings.json

```diff
@@ -25,4 +25,5 @@
   "explorer.compactFolders": false, // present folders in explorer as a tree
   "git.inputValidation": true,
   "git.inputValidationSubjectLength": 50,
+  "i18next.i18nPaths": "/Users/ludvigsvedberg/Programming/dsek/web/src/translations",
```

Comment:
oops

Reply:
I do, however, think this could perhaps be here, just with relative paths?
I don't know how many other people use this extension but it adds this automatically all the time, and I have not found a way of turning it off, which sucks because I have to constantly revert changes to this file.

Reply:
Sounds good. I noticed it only as it was a non-relative path. (I could have been clearer about that)